src/plugins/mer/mergeneraloptionswidget.cpp

```diff
@@ -45,9 +47,13 @@ MerGeneralOptionsWidget::MerGeneralOptionsWidget(QWidget *parent)
     m_ui->environmentFilterTextEdit->setEnabled(!MerSettings::isEnvironmentFilterFromEnvironment());
     m_ui->environmentFilterTextEdit->setToolTip(m_ui->environmentFilterTextEdit->toolTip()
             .arg(Constants::SAILFISH_SDK_ENVIRONMENT_FILTER));
-    m_ui->environmentFilterOverriddenLabel->setVisible(MerSettings::isEnvironmentFilterFromEnvironment());
-    m_ui->environmentFilterOverriddenLabel->setText(m_ui->environmentFilterOverriddenLabel->text()
-            .arg(Constants::SAILFISH_SDK_ENVIRONMENT_FILTER));
+    m_ui->environmentFilterInfoLabel->setVisible(MerSettings::isEnvironmentFilterFromEnvironment());
+    m_ui->environmentFilterInfoLabel->setPixmap(Utils::Icons::WARNING.pixmap());
+    m_ui->environmentFilterInfoLabel->setToolTip(
+            QLatin1String("<font color=\"red\">")
+            + tr("This option is currently overridden with the %1 environment variable.")
+                .arg(Constants::SAILFISH_SDK_ENVIRONMENT_FILTER)
+            + QLatin1String("</font>"));
```

Comment:
It looks like you did a bit more than just shuffle widgets - perhaps the commit message could be improved?

Reply:
I would not complicate it, shuffling is the main point and shuffling one label into icon tooltip is just small part of it :)